config/build.ts

```diff
@@ -430,20 +417,27 @@ function checkSnippetsForUniqueIDs(processedSnippets) {
     }
 }
 
-async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProcessedData>) {
+async function generatePlaylists(processedSnippets: Dictionary<SnippetProcessedData>) {
     banner('Generating playlists');
 
+    let processedPublicSnippets = new Dictionary<SnippetProcessedData>();
+    for (let processedSnippet of processedSnippets.values()) {
+        if (processedSnippet.type === 'public') {
```

Comment:
I much prefer a boolean (or at minimum a strongly-typed string, but really just a boolean), for something like this.  That way, too, you can be guaranteed that the "else" statement will always be hit correctly, as opposed to being hit due to a misspelling.